docs/scripts/buildApi.ts

```diff
@@ -709,8 +709,15 @@ async function updateStylesDefinition(context: {
             if (members) {
               styles.descriptions = styles.descriptions || {};
               members.forEach((member) => {
-                const className = ((member as babel.types.TSPropertySignature)
+                let className = ((member as babel.types.TSPropertySignature)
                   .key as babel.types.Identifier).name;
+
+                if(!className) {
```

Comment:
We need this change in order to support classess that are in kebab-case like `grid-xs-auto` for example